src/v2/Apps/Purchase/Components/OrderRow.tsx

```diff
@@ -38,15 +37,21 @@ const StyledImage = styled(Image)`
   width: 50px;
 `
 
+const UnderlineOnHoverLink = styled(RouterLink)`
```

Comment:
As far as I understood this functionality is not provided out of the box. A bit unclear if this page needs this one off look for links or we need to add this as an option to RouterLink / Link? For now just doing it inline here.

Reply:
Maybe mention this briefly in #design-system?